src/Browser/VersionResolverChrome.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DBrekelmans\BrowserDriverInstaller\Browser;
+
+use DBrekelmans\BrowserDriverInstaller\CommandLine\CommandLineEnvironment;
+use DBrekelmans\BrowserDriverInstaller\Exception\NotImplemented;
+use DBrekelmans\BrowserDriverInstaller\OperatingSystem\OperatingSystem;
+use DBrekelmans\BrowserDriverInstaller\Version;
+use RuntimeException;
+use function Safe\sprintf;
+
+abstract class VersionResolverChrome implements VersionResolver
```

Comment:
I prefer to not have an abstract class, since the implementation between GoogleChrome and Chromium could become different with updates. There's already a difference on MacOS it appears. In this case I think it's easier to maintain if we have 2 implementation classes and duplicate the implementation for now.

Reply:
Done. I wanted to avoid too much code duplication between GoogleChome and Chromium VersionResolver but it is true that it might more difficult to maintain then